site/app/templates/submission/homework/SubmitBox.twig

```diff
@@ -96,22 +96,26 @@
 		            <input type="text" id= "user_id" value ="" placeholder="{{ user_id }}"/>
                 </div>
             </div>
-            <div class="sub hide" id="pdf-submit-button">
+            <div class="sub hide" id="pdf-submit-button"
+            {% if is_bulk_upload %}
+                style="display: revert;"
+            {% endif %}
```

Comment:
This feels pretty hacky.  Couldn't you do the same think by conditionally adding or removing the `hide` class?